src/deletepy/utils/logging_utils.py

```
@@ -0,0 +1,276 @@
+"""Structured logging utilities for DeletePy Auth0 User Management Tool."""
+
+import json
+import logging
+import logging.config
+import os
+import sys
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict, Optional
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)